.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+on:
+  - push
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        node-version:
+          - '12'
+          - '14'
```

Comment:
```suggestion
          - '14'
          - '16'
```

Reply:
or maybe all 3? node@12 EOLs soon enough (2022-04-30) and this is already going to be a major semver update, IMHO just rip the bandaid off now